src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -1111,10 +1114,28 @@ void distributable_compute_gradient(const shared_ptr<ForwardProjectorByBin>& for
                               zero_seg0_end_planes,
                               log_likelihood_ptr,
                               additive_binwise_correction,
-                              /* normalisation info to be ignored */ shared_ptr<BinNormalisation>(), 0., 0.,
-                              &RPC_process_related_viewgrams_gradient,
+                              /* normalisation info to be ignored */ shared_ptr<BinNormalisation>(),
```

Comment:
This is actually incorrect in the case when `log_likelihood_ptr != NULL`. This will need to be corrected in #760 as to compute the log likelihood, you need the normalisation